.github/workflows/master.yaml

```diff
@@ -12,46 +12,13 @@ jobs:
         with:
           java-version: '12.x'
       - uses: subosito/flutter-action@v1
-      #   with:
-      #     channel: 'stable'
-      # - name: Upgrade flutter
-      #   run: |
-      #     flutter channel stable
-      #     flutter upgrade
-      # - name: Install packages
```

Comment:
We are no longer going to perform the flutter upgrade step every run the CI / CD? Currently, it was always pointing to an exact version. 

What reminds me, can I do flutter upgrade?

Reply:
this was commented out code, so this wasnt running. 
I wonder why we commented out this code

Reply:
Yeah, lets upgrade flutter

Reply:
> I wonder why we commented out this code

The CI / CD could not run if it upgraded to null safety and we still did not plan to become a member at that time, so I commented on that part so that it points to the version we used, but now I no longer believe there is an upgrade that has so much impact.

> Yeah, lets upgrade flutter

It will be uncommitted the step then instead of the one that points to a specific version, right?

Reply:
or we hard code the flutter version once we upgrade it?

Reply:
that works too